src/util-ioctl.c

```diff
@@ -25,6 +25,7 @@
 #include "suricata-common.h"
 #include "conf.h"
 #include "util-device.h"
+#include "decode-sll.h"
```

Comment:
this seems out of place here?

Reply:
nm I see it now

Reply:
Needed for `SLL_HEADER_LEN`

Maybe `SLL_HEADER_LEN` can be defined some place else so that util does not depend on decode...
But we may want to do that later